system-test/test.sh

```diff
@@ -63,13 +63,17 @@ node -v
 node --trace-warnings "$BENCHPATH" 10 $VERIFY_TIME_LINE_NUMBERS
 
 if [[ "$VERIFY_TIME_LINE_NUMBERS" == "true" ]]; then
+  pprof -lines -top -nodecount=2 time.pb.gz
```

Comment:
This and subsequent similar lines presumably just print out the pprof output. Is this just to debug the raw output on unexpected behavior?

Reply:
Yes -- this is just to be able to check what pprof prints when the output doesn't match the desired regex.